caboose.go

```diff
@@ -102,27 +102,19 @@ var ErrContentProviderNotFound error = errors.New("saturn failed to find content
 var ErrSaturnTimeout error = errors.New("saturn backend timed out")
 
 type ErrSaturnTooManyRequests struct {
-	Node       string
-	RetryAfter time.Duration // TODO: DRY refactor after https://github.com/ipfs/go-libipfs/issues/188
+	Node string
 }
 
-func (e ErrSaturnTooManyRequests) Error() string {
-	return fmt.Sprintf("saturn node %s returned Too Many Requests error, please retry after %s", e.Node, humanRetry(e.RetryAfter))
+func (e *ErrSaturnTooManyRequests) Error() string {
+	return fmt.Sprintf("saturn node %s returned Too Many Requests error, please retry later", e.Node)
```

Comment:
Note that the actual value will be included on the wrapped error of type `gateway.ErrRetryAfter`.